gallery.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>BrickHack 7</title>
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="keywords" content="hackathons, hackathon, hack-a-thon, programming, hardware, development, design, rit"/>
+
+    <!-- Fonts -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="orange">
+        <div id="rectangle"></div>
+        <h1>BrickHack 6</h1>
+        <div class="info">
```

Comment:
instead of a seperate div with margin, try just putting linebreaks (<br/>) where you want the spaces to be

Reply:
`<br />`